crates/sqlez/Cargo.toml

```diff
@@ -4,6 +4,7 @@ version = "0.1.0"
 edition = "2021"
 publish = false
 license = "GPL-3.0-or-later"
+rust-version = "1.74"
```

Comment:
Do we need that? We have a top-level `rust-toolchain.toml` which kinda handles the MSRV for us and other crates don't seem to have that bit of metadata set.

Reply:
I had overlooked that the version was specified in `rust-toolchain.toml`. Since `as_encoded_bytes` was not available until recently, I put it in there in the hope that it would fail the test or get a comment if there was a problem, and since it doesn't need it, I will remove it.